File: praktikum_algopascal/C/luas_persegi_panjang.c
//luas persegi panjang
#include <stdio.h>



int main()
{
	int p, l;
	int tot;

	printf("Menghitung Luas persegi panjang\n");
	printf("Masukan Nilai P: ");
	scanf("%d", &p);
	printf("Masukan Nilai L: ");
	scanf("%d", &l);

	tot = p * l;

	printf("----------------------------------");
	printf("\nHasil: %d", tot);

	return 0;
}

File: praktikum_algopascal/pascal/luas_persegi_panjang.pas
//luas persegi panjang
program luas;
	uses crt;
		var
			p, l: Integer;
			tot : integer;
	
begin
		writeln('Menghitung Luas Persegi Panjang');
		write('Masukan Nilai P : ');
		readln(p);
		write('Masukan Nilai L : ');
		readln(l);

		tot := p * l;

		writeln('---------------------------------');
		writeln('Hasil : ', tot);
end.